app/controllers/api/v1/portfolio_items_controller.rb

```diff
@@ -3,11 +3,14 @@ module V1
     class PortfolioItemsController < ApplicationController
       include Api::V1::Mixins::IndexMixin
 
-      before_action :write_access_check, :only => %i[create update destroy create_tags]
+      before_action :create_access_check, :only => %i[create]
```

Comment:
Since we now have a separate RBAC action that is only usable on a single method, the `before_action` isn't gaining us anything. These methods could probably just get moved into the `:only` methods declared.

Reply:
@syncrou I think having them as part of the before action shows how easy it is easy to add RBAC in one place, otherwise the RBAC code will be intermingled with the code and will not stand out.